docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,81 @@
+= Enabling a Snyk task in Konflux
+
+[abstract]
+The sast-snyk-check task uses the Snyk Code tool to perform static application security testing (SAST). 
+Specifically, the Snyk check scans an application's source code for potential security vulnerabilities, 
+including SQL injection, cross-site scripting (XSS), and code injection attack vulnerabilities.
+
+> NOTE: You can run a Snyk task only if you have a Snyk token stored in a namespace secret. 
+This must also include the name of your secret in the *snyk-secret* pipeline parameter.
+
+== Enabling Snyk task
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Get a Snyk token.
+.. In the lower left of the home page, click your name, then select *Account* settings.
+.. From the Account Settings page, select *General*, which is the default, then *Auth Token*.
+.. Under the *Auth Token* section, click on *Click to View* button to visualize the *KEY* value of the automatically generated token.
+. To enable Snyk code, go to *Settings* on the left panel > *Snyk Code* > *Enable Snyk Code* > click the *Disabled* bar.
+.. Save the changes when it is changed to *Enabled*.
+. Add the above generated secret *KEY* value to your {ProductName} workspace.
+    .. Login to the {ProductName} UI console https://console.redhat.com/preview/application-pipeline.
+    .. Click on the *Secrets* from the left panel.
+    .. Click on the *Add secret* button.
+    .. Fill out the details on the page appeared.
+    ... Select Secret for *Build*.
+    ... From the *Secret type* dropdown menu, choose *Key/value secret*.
+    ... From the *Secret name* dropdown menu, select *snyk-secret*.
+    ... Under *Key/value secret*, expand *Key/value 1* section the *Key* name is automatically populated as *snyk-token*.
+    ... Paste *snyk-token* value generated from the Snyk account in the above steps into the *Upload the file with value for your key or paste its contents* section.
```

Comment:
--> Paste your Snyk token into the **Upload the file with value for your key or paste its contents** field.

assuming it's a field not a section since the user has to interface there. correct me if i'm wrong!